Cargo.toml

```diff
@@ -20,7 +22,6 @@ crate-type = ["staticlib", "rlib", "cdylib"]
 default = ["ffi", "logger", "zeroize"]
 ffi = ["ffi-support"]
 logger = ["env_logger"]
-vendored = ["openssl", "openssl/vendored"]
```

Comment:
I believe this flag can be preserved as `vendored = ["anoncreds-clsignatures/openssl_vendored"]`